### PR TITLE
Issue #11912: remove all validations during web site build during release

### PR DESCRIPTION
**Pull request by romani**
Issue #11912

see prove that it is ok in issue - https://github.com/checkstyle/checkstyle/issues/11912#issuecomment-1229538320

so we will not have reports in website that nobody care to look and action.
But we will significantly reduce execution time of this release phase.
1 min - without validation
8 min - with validatiion

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ echo "Go to folder where site was build and sources are already at required tag"
 cd target/checkout
 
 echo "Generating web site"
-mvn -e --no-transfer-progress site
+mvn -e --no-transfer-progress site -Pno-validations -Dmaven.javadoc.skip=false
```
Comment (pbludov) [inline release.sh:48]: Javadoc is a part of the site: https://checkstyle.org/apidocs/index.html
The option `maven.javadoc.skip` will disable generation of API reference.

**Review by strkkk — APPROVED** [review]

File: release.sh (hunk 1 above)
Comment (romani) [reply release.sh:48]: It is skip as FALSE, please look at diff of folders, I prove that API is generated.
Content is same, just timestamp are different

**Review by pbludov — APPROVED** [review]

**Review by nrmancuso — APPROVED** [review]

**Review by rnveach — APPROVED** [review]
I think we should still ensure suite passes when doing the release at the start of the process. We don't have some critical need to critically release on time. We should emphasis a product over timeliness.

**Comment by romani** [comment]
It is too late to care about our over care of quality in each symbol of code.
If leak happened it just should be fixed in next commit, actual user of our product are not affected by our slightly damaged code quality.

In absolutely worse case, we will do one more release.

**Pull request merged**